tap-xunit/plan.sh

```diff
@@ -0,0 +1,26 @@
+pkg_name=tap-xunit
+pkg_origin=core
+pkg_version=2.3.0
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_license=("MIT")
+pkg_source="https://github.com/aghassemi/tap-xunit/archive/${pkg_version}.tar.gz"
+pkg_shasum=2ab6d5bbe47cc01cc12bbb8df6e3c0f1f24b8b9b17c52cad246a291f7468612f
+pkg_deps=(
+  core/coreutils
+  core/node
+)
+pkg_bin_dirs=(bin)
+pkg_description="TAP to xUnit XML converter"
+pkg_upstream_url=https://github.com/aghassemi/tap-xunit
+
+do_build() {
+  return 0
+}
+
```

Comment:
I was going to ask if it was possible to add the project's [test command `node test/runner.js && tape unit-tests/**/*.js`](https://github.com/aghassemi/tap-xunit/blob/master/package.json#L11) as a do_check but it looks like the project's dependencies are not installed at the point in time that `do_check` happens.
It looks non-trivial to get the tests working in do_check, but then again I have 0 npm experience.